internal/iothread/iothread.go

```diff
@@ -2,34 +2,12 @@ package iothread
 
 import (
 	"context"
-	"errors"
-	"fmt"
 	"log/slog"
-	"net"
-	"strconv"
-	"sync/atomic"
-	"syscall"
-	"time"
 
-	"github.com/dicedb/dice/config"
 	"github.com/dicedb/dice/internal/auth"
-	"github.com/dicedb/dice/internal/clientio"
 	"github.com/dicedb/dice/internal/clientio/iohandler"
-	"github.com/dicedb/dice/internal/clientio/requestparser"
-	"github.com/dicedb/dice/internal/cmd"
-	diceerrors "github.com/dicedb/dice/internal/errors"
-	"github.com/dicedb/dice/internal/ops"
-	"github.com/dicedb/dice/internal/querymanager"
-	"github.com/dicedb/dice/internal/shard"
-	"github.com/dicedb/dice/internal/wal"
-	"github.com/dicedb/dice/internal/watchmanager"
-	"github.com/google/uuid"
 )
 
-const defaultRequestTimeout = 6 * time.Second
-
-var requestCounter uint32
-
 // IOThread interface
 type IOThread interface {
```

Comment:
I like the `Handler` naming convention. It follows Go naming convention, where the data-structure names are encouraged to be verbs name. Based on this should we rename IOThread to IOHandler?

Reply:
#1394